domain/ide/converter/converter.go

```diff
@@ -159,25 +161,225 @@ func ToDiagnostics(issues []snyk.Issue) []types.Diagnostic {
 	// Sending null instead of an empty array leads to stored diagnostics not being cleared.
 	// Do not prefer nil over an empty slice in this case. The next line ensures that even if issues is empty,
 	// the return value of this function will not be null.
-	diagnostics := []types.Diagnostic{}
 
-	for _, issue := range issues {
+	var diagnostics []types.Diagnostic
```

Comment:
Wondering: Is this empty array or nil?

Reply:
According to [this](https://www.perplexity.ai/search?q=pending&newFrontendContextUUID=bb514303-e101-4531-a6d5-99fd11f32b2a), it should stay like it was.

Reply:
it's nil. but append can append to nil.

according to this it doesn't make a difference
https://stackoverflow.com/a/29164565